test/spoom/sorbet/translate/sorbet_sigs_to_rbs_comments_test.rb

```diff
@@ -288,6 +288,54 @@ def foo
           RBS
         end
 
+        def test_translate_to_rbs_with_nested_nilable_param
+          contents = <<~RB
+            Class.new do
+              sig { params(x: T.nilable(T.nilable(Integer))).void }
```

Comment:
Is our logic for this purpose-built for RBS? This kind of normalization would be useful more generally. Would be nice to do: `sig -> normalized sig -> RBS`